src/vector_index.h

```diff
@@ -17,6 +17,9 @@
 #define VECSIM_M "M"
 #define VECSIM_EF "EF"
 
+#define BASE64_ENABLED 1
+#define BASE64_DISABLED 2
+
```

Comment:
other values possible? is this a bool?

Reply:
Yes.
For the Phonetic attribute, there are `define`s so I followed in the same footsteps.

Reply:
I meant, logically. Can it have other values?

Reply:
Not that I can think of. We might opt out for a different solution than base64 but for now, this is the path we chose.